string save_path
string image_topic_name
---
---
